doc/source/release-actions/index.rst

```diff
@@ -31,3 +31,53 @@ Release GitHub action
 .. jinja:: release-github
     :file: _templates/action.rst.jinja
 
+Release PyPI template
+---------------------
+
+Below, it is assumed that a job `build-library` already created library sources
+and wheel artifacts using the environment variable `PACKAGE_NAME`. Also, the job
+is defined to not upload files if one already exists thanks to the
+`skip-existing` input.
+
+.. code::yaml
+
+    release-pypi:
+    name: "Release to PyPI with trusted publisher approach"
+    runs-on: ubuntu-latest
+    needs: [build-library]
+    # Specifying a GitHub environment is optional, but strongly encouraged
+    environment: release
+    permissions:
+        # IMPORTANT: this permission is mandatory for trusted publishing
+        id-token: write
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+    steps:
+        - name: "Download the library artifacts from build-library step"
+        uses: actions/download-artifact@v4
+        with:
+            name: ${{ env.PACKAGE_NAME }}-artifacts
+            path: ${{ env.PACKAGE_NAME }}-artifacts
+
+        - name: "Display the structure of downloaded files"
+        shell: bash
+        run: ls -R
+
+        - name: "Upload artifacts to PyPI using Trusted Publisher"
+        uses: pypa/gh-action-pypi-publish@v1.12.4
+        with:
+            repository-url: "https://upload.pypi.org/legacy/"
+            print-hash: true
+            packages-dir: ${{ env.PACKAGE_NAME }}-artifacts
+            skip-existing: false
+
+When used to test the release process, you can update the repository URL as
+follow:
```

Comment:
```suggestion
When testing the release process, you can update the repository URL as
follows:
```
I hope I didn't change the meaning, but the subject "you" isn't related to "When used to test the release process, ...." For the existing construction, you would need to have something like this: "When used to test the release process, the following action updates the repository URL."